manifest.yml

```diff
@@ -33,3 +33,9 @@ applications:
       instances: ((worker_instances))
       command: yarn start:worker
       memory: ((worker_memory))
+  - name: tta-smarthub-maintenance-page-((env))
+    buildpacks:
+      - staticfile_buildpack
+    root: maintenance_page
+    force_https: "true"
```

Comment:
I don't think we'd need `force_https` (unless that's explicitly in the cloud.gov docs) and I think you should specify the memory quota for this app. I think 512M is the default and I can't imagine the `staticfile_buildpack` would need that.